src/usage_metrics/helpers.py

```diff
@@ -154,3 +156,28 @@ def get_table_name_from_context(context: OutputContext) -> str:
     if context.has_asset_key:
         return context.asset_key.to_python_identifier()
     return context.get_identifier()
+
+
+def make_request(
+    url: str, headers: str | None = None, params: str | None = None, timeout: int = 100
+) -> requests.models.Response:
+    """Makes a request to the github api.
```

Comment:
This function doesn't have anything GH-specific, right? It's just a thin wrapper around `requests.get` that prints out the response text on error?

It could almost make sense to just use `requests.get` at the call sites without the extra level of indirection - that, or renaming this function to make it clear that it's a general GET request sender with special error handling behavior.

Reply:
Eventually I'd like to add some `retry` functionality here, which was part of the incentive to add a layer of abstraction. Which is out of scope for this PR, of course! But updated the docs to reflect that it's not Github specific, just a relic of where this function was initially taken from.